RELEASE.md

```diff
@@ -0,0 +1,26 @@
+# Release Process
+
+## Overview
+
+At any given time there are three versions of the Prometheus charm
+available
+
+1. Stable - This is a well tested production ready version of the
```

Comment:
Should this read `latest/stable` (and similarly `latest/edge`, `latest/beta`) to match up with Charmhub?

Reply:
Thanks. Changed accordingly.